node/v2/error_response.js

```diff
@@ -66,78 +67,113 @@ CodeNames[Codes.ProtocolError] = 'protocol error';
 CodeNames[Codes.Unhealthy] = 'unhealthy';
 
 var CodeErrors = Object.create(null);
-CodeErrors[Codes.Timeout] = TypedError({
-    type: 'tchannel.timeout',
-    message: 'TChannel timeout',
-    isErrorFrame: true,
-    codeName: 'Timeout',
-    errorCode: Codes.Timeout,
-    originalId: null
-});
-CodeErrors[Codes.Cancelled] = TypedError({
-    type: 'tchannel.cancelled',
-    message: 'TChannel cancelled',
-    isErrorFrame: true,
-    codeName: 'Cancelled',
-    errorCode: Codes.Cancelled,
-    originalId: null
-});
-CodeErrors[Codes.Busy] = TypedError({
-    type: 'tchannel.busy',
-    message: 'TChannel busy',
-    isErrorFrame: true,
-    codeName: 'Busy',
-    errorCode: Codes.Busy,
-    originalId: null
-});
-CodeErrors[Codes.Declined] = TypedError({
-    type: 'tchannel.declined',
-    message: 'TChannel declined',
-    isErrorFrame: true,
-    codeName: 'Declined',
-    errorCode: Codes.Declined,
-    originalId: null
-});
-CodeErrors[Codes.UnexpectedError] = TypedError({
-    type: 'tchannel.unexpected',
-    message: 'TChannel unexpected error',
-    isErrorFrame: true,
-    codeName: 'UnexpectedError',
-    errorCode: Codes.UnexpectedError,
-    originalId: null
-});
-CodeErrors[Codes.BadRequest] = TypedError({
-    type: 'tchannel.bad-request',
-    message: 'TChannel bad request',
-    isErrorFrame: true,
-    codeName: 'BadRequest',
-    errorCode: Codes.BadRequest,
-    originalId: null
-});
-CodeErrors[Codes.NetworkError] = TypedError({
-    type: 'tchannel.network',
-    message: 'TChannel network error',
-    isErrorFrame: true,
-    codeName: 'NetworkError',
-    errorCode: Codes.NetworkError,
-    originalId: null
-});
-CodeErrors[Codes.ProtocolError] = TypedError({
-    type: 'tchannel.protocol',
-    message: 'TChannel protocol error',
-    isErrorFrame: true,
-    codeName: 'ProtocolError',
-    errorCode: Codes.ProtocolError,
-    originalId: null
-});
-CodeErrors[Codes.Unhealthy] = TypedError({
-    type: 'tchannel.unhealthy',
-    message: 'TChannel unhealthy',
-    isErrorFrame: true,
-    codeName: 'Unhealthy',
-    errorCode: Codes.Unhealthy,
-    originalId: null
-});
+CodeErrors[Codes.Timeout] = function TChannelTimeoutError(originalId, message) {
+    LiteError.call(this);
+    this.type = this.fullType = 'tchannel.timeout';
+    this.message = message || 'TChannel timeout';
+    this.isErrorFrame = true;
+    this.codeName = 'Timeout';
+    this.errorCode = Codes.Timeout;
+    this.originalId = originalId;
+    this.name = 'TchannelTimeoutError';
+};
+util.inherits(CodeErrors[Codes.Timeout], LiteError);
+
+CodeErrors[Codes.Cancelled] = function TChannelCancelledError(originalId, message) {
+    LiteError.call(this);
+    this.type = this.fullType = 'tchannel.cancelled';
+    this.message = message || 'TChannel cancelled';
```

Comment:
message wasn't variable before, was it? if it is meant to be now, please also push this field down to last / second-to-last